components/browser/thumbnails/src/main/java/mozilla/components/browser/thumbnails/storage/ThumbnailStorage.kt

```diff
@@ -77,11 +79,13 @@ class ThumbnailStorage(
      * Stores the given thumbnail [Bitmap] into the disk cache with the provided session ID or url
      * as its key.
      */
-    fun saveThumbnail(sessionIdOrUrl: String, bitmap: Bitmap) {
-        logger.debug(
-            "Saved thumbnail to disk (sessionIdOrUrl = $sessionIdOrUrl, " +
-                "generationId = ${bitmap.generationId})"
-        )
-        sharedDiskCache.putThumbnailBitmap(context, sessionIdOrUrl, bitmap)
-    }
+    @WorkerThread
```

Comment:
With this now doing the work asynchronously this doesn't seem to need a `WorkerThread` annotation? 🤔

Reply:
Ah, yeah, that doesn't help here. Removed.